core/web/resolver/eth_key.go

```diff
@@ -52,15 +52,22 @@ func (r *ETHKeyResolver) IsFunding() bool {
 	return r.key.state.IsFunding
 }
 
-func (r *ETHKeyResolver) ETHBalance() *string {
+func (r *ETHKeyResolver) ETHBalance(ctx context.Context) *string {
 	if r.key.chain == nil {
 		return nil
 	}
 
 	balanceMonitor := r.key.chain.BalanceMonitor()
 
 	if balanceMonitor == nil {
-		return nil
+		balance, err := r.key.chain.Client().BalanceAt(ctx, r.key.state.Address.Address(), nil)
```

Comment:
I don't think we want to do this since it makes rendering very expensive if balance monitor is disabled. Makes more sense to me to simply render nil

Reply:
Resolved